static/src/js/components/SavedProjects/__tests__/SavedProjects.test.jsx

```diff
@@ -1,142 +1,130 @@
 import React from 'react'
-import Enzyme, { shallow } from 'enzyme'
-import Adapter from '@wojtekmaj/enzyme-adapter-react-17'
-import Table from 'react-bootstrap/Table'
+import { screen } from '@testing-library/react'
 
-import Spinner from '../../Spinner/Spinner'
-import PortalLinkContainer from '../../../containers/PortalLinkContainer/PortalLinkContainer'
 import { SavedProjects } from '../SavedProjects'
-
-Enzyme.configure({ adapter: new Adapter() })
-
-function setup(overrideProps) {
-  const props = {
-    savedProjects: [],
-    savedProjectsIsLoading: false,
-    savedProjectsIsLoaded: false,
+import setupTest from '../../../../../../jestConfigs/setupTest'
+
+jest.mock('../../../util/addToast', () => ({
+  addToast: jest.fn()
+}))
+
+jest.mock(
+  '../../../containers/PortalLinkContainer/PortalLinkContainer',
+  () => ({
+    __esModule: true,
+    default: ({ to, onClick, children }) => (
+      <a href={to} onClick={onClick}>
+        {children}
+      </a>
+    )
+  })
+)
+
+const setup = setupTest({
+  Component: SavedProjects,
+  defaultProps: {
+    projects: [],
+    isLoading: false,
+    isLoaded: false,
+    earthdataEnvironment: 'uat',
     onChangePath: jest.fn(),
-    onDeleteSavedProject: jest.fn(),
-    ...overrideProps
-  }
-
-  const enzymeWrapper = shallow(<SavedProjects {...props} />)
-
-  return {
-    enzymeWrapper,
-    props
+    onDeleteProject: jest.fn()
   }
-}
+})
 
 describe('SavedProjects component', () => {
-  describe('when the projects are loading', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
```

Comment:
This is unnecessary, I recently added `clearMocks: true` to the jest config, it does the same thing, but for every test, https://jestjs.io/docs/configuration#clearmocks-boolean